\nSimulated Annealing...Discreted\n

Diagnostic information.
	objective function = @sa_cav_y_4dof
	X0 = [  20 140 1 1.48 ]
Modified options:
	options.AnnealingFcn = @d_annealing_c4
	options.TemperatureFcn = @tempboltzexp
	options.StallIterLimit = 1000
	options.InitialTemperature = [  100 100 100 100 ]
	options.Display = 'diagnose'
	options.ReannealInterval = 50
	options.DisplayInterval = 1 
	options.AcceptanceFcn = @acceptancesa
End of diagnostic information.


                           Best        Current           Mean
Iteration   f-count         f(x)         f(x)         temperature
     0          1        4.50127        4.50127            100

 Iter 0 | Best F(x) 4.501 | F(x)  4.501 | Temperatura 100.000 | k = 1
 
 Iter 1 | Best F(x) 1.000 | F(x)  1.000 | Temperatura 
	Valor atual:H/L = 20.000 t1dt0 = 140.000 L1/L0 1.000 alpha 1.480
	 Novos valores: H/L = 1.000 t1dt0 = 17.000  L1/L0 0.100 alpha 1.380
-------------
{Reference to non-existent field 'InitialTemperatures'.

Error in tempboltzexp (line 19)
init_temp = options.InitialTemperatures

Error in saupdates (line 23)
    solverData.temp = max(eps,options.TemperatureFcn(optimvalues,options));

Error in saengine (line 35)
    solverData = saupdates(solverData,problem,options);

Error in simulanneal (line 51)
solverData = saengine(solverData,problem,options);

Error in simulannealbnd (line 122)
[x, fval, exitflag, output] = simulanneal(FUN, x0, [], [], [], [], lb, ub, options);

Error in d_sa_cav_y4 (line 20)
[x,fval,exitflag,output] = ...

Error in dopt_sa_cavy_4 (line 109)
vetor = d_sa_cav_y4(fun,x0,d,s,rs,optValue,modeltemp);

Caused by:
    Failure in TemperatureFcn evaluation.
} 
dopt_sa_cavy_4(.3,.05,@tempboltzexp)

x0 =

   20.0000   80.0000    0.0700    1.1900


Geometria invï¿½lida para x0 [ 20.000 80.000 0.070 1.190]
